settings.gradle

```diff
@@ -0,0 +1,11 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+    }
+}
+
+rootProject.name = 'JsonConfiguration'
```

Comment:
Don't we need to change this for the maven repo? `json-configuration` would be the preference.

Reply:
Nah, the issue was only with github pacakges, so we can just keep as it is bcu we directly uploading to Reposilite.